samples: fast_pair: locator_tag: Correct reference board readme entries (#28041)
nrfconnect/sdk-nrf: samples/bluetooth/fast_pair/locator_tag/README.rst

              Enables the DFU mode for five minutes.

>    .. group-tab:: Thingy:53 and nRF54L15 TAG

Reviewer: we called this tab "Reference board targets" in the Configuration section. Can we do the same here?

```suggestion
   .. group-tab:: Reference board targets
```

      RGB LED:
         +-------------------------+------------------------+
         | Reference board targets | RGB LED on-board label |
         +=========================+========================+

>          | Thingy:53               | ``LD1``                |

Reviewer 2: ```suggestion
         | Thingy:53               | ``LED1``                |
```

Reviewer: The label is LD1:

https://docs.nordicsemi.com/bundle/ug_thingy53/page/UG/thingy53/hw_description/leds_buttons.html#ariaid-title2

_The Nordic Thingy:53 is equipped with a single RGB LED (LD1) that is controlled (...)_

         +-------------------------+------------------------+
         | nRF54L15 TAG            | ``LED1``               |
         +-------------------------+------------------------+

>          The board displays the application state in the RGB scale using on-board RGB LED.

Reviewer 2: ```suggestion
         The board displays the application state in the RGB scale using an onboard RGB LED.
```
